source/fundamentals/geo.txt

```diff
@@ -0,0 +1,332 @@
+.. _golang-geo:
+
+=========================
+Work with Geospatial Data
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to work with **geospatial data**; data formats,
+indexes, and queries. Geospatial
+data represents a geographic location on the surface of the Earth, or data
+on a Euclidean plane.
+
+Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+- Points on a graph
+
+Store Geospatial Data
+---------------------
+
+You have two options to store geospatial data in MongoDB:
+
+- GeoJSON, a format that represents geospatial data on an earth-like
+  sphere.
+
+- Legacy Coordinate Pair, a format that represents geospatial data
+  on a Euclidean plane.
+
+GeoJSON
+~~~~~~~
+
+GeoJSON is the preferred format to store geospatial data in MongoDB.
+Specifying a location begins with declaring one or more **positions**,
+and choosing the appropriate **type**.
+
+GeoJSON Positions
+`````````````````
+
+A position represents a single place on Earth and exists in code as an array
+containing the following values:
+
+- Longitude in the first position (required)
+- Latitude in the second position (required)
+- Elevation in the third position (optional)
+
+The following is the **position** of the MongoDB Headquarters in
+New York City, NY.
+
+.. code-block:: go
+   :copyable: False
+
+   []float64{-73.986805, 40.7620853}
+
+.. important:: Longitude then Latitude
+
+  GeoJSON orders coordinates as **longitude** first and **latitude** second.
+  This may be surprising as geographic coordinate system conventions generally list
+  latitude first and longitude second. Make sure to check what format any other
+  tools you are working with use. Popular tools such as OpenStreetMap and Google
+  Maps list coordinates as latitude first and longitude second.
+
+
+GeoJSON Types
+`````````````
+
+Your GeoJSON object's type determines its geometric shape. Geometric shapes are
+made up of positions.
+
+Here are some common GeoJSON types and how you can specify them with positions:
+
+- ``Point``: a single position. This could represent the location of the
+  MongoDB Headquarters.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "HQ"},
+         {"location", bson.D{
+             {"type", "Point"},
+             {"coordinates", []float64{-73.986805, 40.7620853}},
+         }},
+     }
```

Comment:
I'm not sure this adds value, and in later types the format is different because of the shape requirements.

Reply:
The value this intends to add is to help further distinguish a "Point" from a "Position". AKA You use a Position to construct a Point.

The meaning of point + position tripped up Mike, who is learning this for the first time while reading it, and we agreed that helping to further clarify the difference here could be helpful.

This isn't super important, so I will not block a merge because of this.

Reply:
I updated the text to capture the intent of your suggestion.